docs/index.md

```diff
@@ -1,241 +1,172 @@
-```
-
-1. The **Pydantic Logfire** SDK can be installed from PyPI or Conda, [Learn more](guides/first_steps/index.md#install).
-2. The SDK comes with a CLI for authentication and more, [Learn more](reference/cli.md).
-
-Then in your code:
-
-```py
-import logfire
-from datetime import date
-
-logfire.configure()  # (1)!
-
-logfire.info('Hello, {name}!', name='world')  # (2)!
-
-with logfire.span('Asking the user for their {question}', question='birthday'):  # (3)!
-    user_input = input('When were you born [YYYY-mm-dd]? ')
-    dob = date.fromisoformat(user_input)  # (4)!
-    logfire.debug('{dob=} {age=!r}', dob=dob, age=date.today() - dob)  # (5)!
-```
-
-1. This should be called once before logging to initialize Logfire. If no project is configured for the current directory, an interactive prompt will walk you through creating a project.
-2. This will log `Hello world!` with `info` level. `name='world'` will be stored as an attribute that can be queried with SQL.
-3. Spans allow you to nest other Logfire calls, and also to measure how long code takes to run. They are the fundamental building block of traces!
-4. Attempt to extract a date from the user input. If any exception is raised, the outer span will include the details of the exception.
-5. This will log for example `dob=2000-01-01 age=datetime.timedelta(days=8838)` with `debug` level.
+=== "Production"
+    !!! tip "Production setup"
+        In production, we recommend you provide your write token to the Logfire SDK via environment variables.
 
-This might look similar to simple logging, but it's much more powerful — you get:
+    Production tab
 
-- structured data from your logs
-- nested logs / traces to contextualize what you're viewing
-- a custom-built platform to view your data, with no configuration required
-- and more, like pretty display of Python objects — see below
 
-!!! note
-    If you have an existing app to instrument, you'll get the most value out of [configuring OTel integrations](#otel), before you start adding `logfire.*` calls to your code.
-
-![Logfire hello world screenshot](images/index/logfire-screenshot-hello-world-age.png)
-
-### Python and Pydantic insights :snake: {#python}
-
-From rich display of Python objects to event-loop telemetry and profiling Python code, Pydantic Logfire can give you a clearer view into how your Python is running than any other observability tool.
 
-Logfire also has an out-of-the-box Pydantic integration that lets you understand the data passing through your Pydantic models and get analytics on validations.
+## Tracing with Spans
 
-We can record Pydantic models directly:
+Spans let you add context to your logs and measure code execution time. Multiple spans combine to form a trace,
+providing a complete picture of an operation's journey through your system.
 
 ```py
-from datetime import date
+from pathlib import Path
 import logfire
-from pydantic import BaseModel
+
+cwd = Path.cwd()
+total_size = 0
 
 logfire.configure()
```

Comment:
`logfire.configure()` should come immediately after imports.

Reply:
This section is on a new page now, fixed there. `/docs/get-started/traces`